### DISCUSSION TITLE
[@types/node] 16.x AbortSignal missing all methods to satisfy Web API as well as inherited methods from EventTarget to also satisfy Node spec

### DISCUSSION THREAD

**Discussion by loucadufault** [DefinitelyTyped/DefinitelyTyped#66556] [Issues with a @types package]
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/v16/globals.d.ts#L47

Should conform to https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal

Nodejs 16.x (LTS) defines the AbortSignal as https://nodejs.org/docs/latest-v16.x/api/globals.html#class-abortsignal. The node types module is missing the inherited methods from EventTarget.

This is the case in Typescript lib declarations: https://github.com/microsoft/TypeScript/blob/v4.6.4/lib/lib.dom.d.ts#L1913-L1921

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
Added in PR #68227 and #68250
